src/containers/other/NFTView/elements/DomainSettings/elements/DomainSettingsBody/hooks/useDomainSettingsBodyData.tsx

```diff
@@ -18,8 +18,8 @@ export const useDomainSettingsBodyData = (
 			'';
 		const story = metadata?.description || '';
 		const isMintable = Boolean(metadata?.isMintable);
-		const isBiddable = Boolean(metadata?.isBiddable);
-		const gridViewByDefault = Boolean(metadata?.gridViewByDefault);
+		const isBiddable = true;
+		const gridViewByDefault = true;
```

Comment:
This will break if metadata has `isBiddable: false` or `gridViewByDefault: false` in metadata. These lines should be able to stay as is, and defaults should be set in `metadata.ts:parseMetadata`

Reply:
Makes sense thanks. Reverted this change however, the settings won't be toggled as these return false.